README.md

```diff
@@ -19,3 +19,28 @@ Project members can directly update this repository.
 1. Create a feature branch using the following naming schema: *your_name*/*short_description*
 2. Push your branch to the repository and make a pull request to merge into the main branch
 3. Assign at least one reviewer and one person to handle the merge request. 
+
+## Setup docs and view documentation locally
+
+1. Create conda environment if desired:
+
+``` bash
+conda create --name IMPROVE python=3.9.16
+```
+
+2. Get docs repo and install dependencies:
+
+``` bash
+git clone git@github.com:JDACS4C-IMPROVE/docs.git
```

Comment:
Change to 
`git clone https://github.com/JDACS4C-IMPROVE/docs.git`

Previous command didn't work for me. Got this message:

```
Warning: Permanently added 'github.com,140.82.113.4' (ECDSA) to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

```

Reply:
@jonesse3  If you are using the https version, how do you push changes back?

Reply:
OK, then ignore this. I just need to fix my setup.

Reply:
I can push changes with the https version with my username and password (personal access token).